components/import-action/menu/menu.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Icon } from "@teambit/evangelist.elements.icon";
+import { Registry, Import, Install } from "../tab-content";
+import { Tab } from "../tab";
+import styles from "./menu.module.scss";
+
+export type MenuProps = {
+  /**
+   * package link to be copied
+   */
+  packageLink: string;
+  /**
+   * import link to be copied
+   */
+  bitLink: string;
+  /**
+   * registry link to be copied
+   */
+  registryName: string;
+  /**
+   * component name to be presented
+   */
+  componentName: string;
+  /**
+   * currently active tab
+   */
+  activeTab: string;
+  /**
+   * change currently active tab
+   */
+  setActiveTab: (active: string) => void;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+export function Menu({
+  packageLink,
+  bitLink,
+  registryName,
+  componentName,
+  activeTab,
+  setActiveTab
+}: MenuProps) {
+  if (activeTab === "registry") {
```

Comment:
why is this outside?

Reply:
because it changes the entire content as opposed to the other ones